influxql/ast.go

```diff
@@ -2040,6 +2042,17 @@ func (s *ShowStatsStatement) RequiredPrivileges() ExecutionPrivileges {
 	return ExecutionPrivileges{{Admin: true, Name: "", Privilege: AllPrivileges}}
 }
 
+// ShowShardGroupsStatement represents a command for displaying groups of shard in the cluster.
```

Comment:
Just say "shard groups" instead of "groups of shard". "shard groups" has a specific meaning.